README.md

```diff
@@ -19,8 +19,25 @@ The package can be installed from github with:
 remotes::install_github("AlexsLemonade/scpcaTools")
 ```
 
-
 ## Testing
 
 To test the import functions in this package, you must also have installed the [`scpcaData` package](https://github.com/AlexsLemonade/scpcaData).
 This can be done by running `remotes::install_github("AlexsLemonade/scpcaData")`
+
+## Docker image and `renv`
+
+This repository also includes a Dockerfile and associated scripts for building a Docker image that includes the `scpcaTools` package and some additional packages used in other ScPCA repositories, notably [scpca-nf](https://github.com/AlexsLemonade/scpca-nf).
+To support this, we use [`renv`](https://rstudio.github.io/renv/index.html) to track the versioned set of packages that will be installed in that Docker image.
+
+Packages that are required for the main `scpcaTools` package should be included in the `renv.lock` file as they are installed and used.
```

Comment:
Just to be clear if you were to want to add a package to `scpcaTools`, you would need to install it while in the R project and add it to the package using `usethis()` before running `renv::snapshot()`? If so, maybe including a sentence here that explicitly states these steps might be beneficial.

Reply:
Adding to the package with `usethis()` is not required: `renv` will scan all R code, so it should find things whether or not they are in the DESCRIPTION file. In particular, it will find things for the test functions, whether or not they are required for the main package. Also while all of the packages do need to be installed, they may or may not need to be mentioned specifically: dependencies may not be explicitly mentioned, but will appear in the lockfile. 

So yeah, this is a bit vague, but I'm not quite sure what to do about it, without going into a lot of depth...

Reply:
Ah okay. I think the instructions you have are good then, I wouldn't go too much more in detail to avoid the risk of being more confusing.